src/Compilers/Core/Portable/CodeGen/ILBuilderEmit.cs

```diff
@@ -110,6 +110,18 @@ internal void EmitArrayBlockInitializer(ImmutableArray<byte> data, SyntaxNode sy
             EmitToken(initializeArray, syntaxNode, diagnostics);
         }
 
+        internal void EmitCreateSpan(ImmutableArray<byte> data, Cci.IMethodReference createSpanHelper, SyntaxNode syntaxNode, DiagnosticBag diagnostics)
```

Comment:
>EmitCreateSpan [](http://example.com/codeflow?start=22&length=14)

It is not clear why this helper is placed in this type. It looks like it is used only by ```CodeGenerator```. Consider moving it there, i.e. closer to where it is used.